backend-services/src/web_asgi/main.py

```diff
@@ -59,6 +92,31 @@ async def post_bookmark_create(request: CreateBookmark) -> Bookmark:
     return await create_bookmark(mongo_engine, request)
 
 
+@app.post("/otp/limit/create", response_model=None, status_code=status.HTTP_201_CREATED)
```

Comment:
I think there should be a PUT request for users who want to update their limit's threshold.